### ISSUE TITLE
BUG: Faltan Casas de cambio

### ISSUE THREAD

**Issue by sequielo** [eduair94/cambio-uruguay#1]
A este momento https://cambio-uruguay.com/ solo lista Prex como única casa de cambio.

![image](https://github.com/eduair94/cambio-uruguay/assets/1847718/4ca9d9d1-9040-4b6f-b927-b71b3d28f0b3)

Investigando el código y probando localmente, "La favorita" está interrumpiendo el proceso de parseo en las demás.
O sea, comentando está linea https://github.com/eduair94/cambio-uruguay/blob/main/classes/origins.ts#L48 permite que la sincronización continue.

Muestro captura desde mi local:

![image](https://github.com/eduair94/cambio-uruguay/assets/1847718/5de17b1e-7b98-4056-b880-74d234f0c883)

Si es posible arreglarlo, el proceso de sync debería ignorar aquellas que fallan y seguir con las demás normalmente.

Más allá de ello, muy buen proyecto! Felicitaciones!

**Comment by eduair94** [comment]
Sí, vi el problema. Como estuve enfermo bastante rato. Pero ya quedó arreglado entiendo. Estoy viendo de actualizar nuevos cambios que están dando problemas con datos obsoletos. Gracias por el aviso.
